roles/setup_sushy_tools/vars/main.yml

```diff
@@ -0,0 +1 @@
+sushy_x86_64_ovmf_code_path: "/usr/share/edk2/ovmf/OVMF_CODE.secboot.fd"
```

Comment:
I think you can put it in the defaults directory

Reply:
I originally had it in defaults and moved it because it felt a little dirty to me.  Correct me if I'm wrong but defaults feels like it should be, well, a default and not then overwritten by the playbook logic.

Let me know what you decide and if needed I can move it to defaults.

Reply:
to clarify, defaults to me says the user can pass in a different path, but it could still be overwritten by the logic.

Reply:
if it's all internal logic, why put it anywhere in defaults or vars? just handle it all in the task itself imo

Reply:
ok - I put all the logic in the tasks

Reply:
@p3ck, you bring up an interesting topic. The vars file will always overwrite the defaults file in term of precedence. That being said vars files should only be used by users looking for overrides. What do you think?

Reply:
I really don't understand the logic behind ansible sometimes. :-)

I would have expected vars to be private to the role and defaults to be sane defaults that the user should be ok with overriding.

This article seems sane to me.
https://www.bouledef.eu/~tleguern/articles/ansible-private-variables/en/

Any private vars should be prepended with __ to signify that they are not to be overridden.  We have also prepended variables with the role name to keep from clashing with other roles vars.  so something like this would make sense to me:

__<ROLENAME>_<VARNAME>

This is just a suggestion.  I'm not bothered too much.  I hope the updated PR here is satisfactory.